modules/ROOT/pages/_partials/cookbook-dw/rename-keys-ex02/transform.dwl

```diff
@@ -0,0 +1,13 @@
+%dw 2.0
+output application/json
+---
+payload.flights map (flight) ->
+flight mapObject (value, key) -> {
+ ((key) match
```

Comment:
I would remove the parenthesis around key

Reply:
revised with example provided by Shoki on Slack